### DISCUSSION TITLE
Configuration error - session start error

### DISCUSSION THREAD

**Discussion by VSnoereJ** [SafeExamBrowser/seb-win-refactoring#587] [General]
A student can't start SEB when clicking the link provided by the teacher. All other students from that class are able to open the test.

Already tried to reinstall SEB, but without succes. 

Thanks in advance!

![MicrosoftTeams-image (1)](https://user-images.githubusercontent.com/107060643/226875199-cbeaf1e8-e4fb-4854-ab9b-0b45aaa39c64.png)

![MicrosoftTeams-image](https://user-images.githubusercontent.com/107060643/226875208-0248e9ce-b2f6-4443-949e-a79f9d74f646.png)

[sebwindowsservice.log](https://github.com/SafeExamBrowser/seb-win-refactoring/files/11038927/sebwindowsservice.log)

**Comment by dbuechel** [comment]
Please provide the log file mentioned in the error message. The one you've attached is from the legacy version 2.x.
